Controller/UserController.php

```diff
@@ -37,6 +40,103 @@ public function indexAction()
+
+        /** @var WysiwygProfileRepository $repository */
+        $repository = $em->getRepository('EMSCoreBundle:WysiwygProfile');
+        $result = $repository->findBy([], ['orderKey' => 'asc'], 1);
+        if(count($result) > 0){
+            $user->setWysiwygProfile($result[0]);
+        }
+
+
+
+        $form = $this->createFormBuilder($user)
+            ->add('username', null, array('label' => 'form.username', 'translation_domain' => 'FOSUserBundle'))
+            ->add('email', EmailType::class, array('label' => 'form.email', 'translation_domain' => 'FOSUserBundle'))
+            ->add('plainPassword', RepeatedType::class, array(
+                'type' => PasswordType::class,
+                'options' => array('translation_domain' => 'FOSUserBundle'),
+                'first_options' => array('label' => 'form.password'),
+                'second_options' => array('label' => 'form.password_confirmation'),
+                'invalid_message' => 'fos_user.password.mismatch',))
+
+            ->add('allowedToConfigureWysiwyg', CheckboxType::class, [
+                'required' => false,
+            ])
+            ->add('wysiwygProfile', EntityType::class, [
+                'required' => false,
+                'label' => 'WYSIWYG profile',
+                'class' => 'EMSCoreBundle:WysiwygProfile',
+                'choice_label' => 'name',
+                'query_builder' => function (EntityRepository $er) {
+                    return $er->createQueryBuilder('p')->orderBy('p.orderKey', 'ASC');
+                },
+            ])
+            ->add('wysiwygOptions', TextareaType::class, [
+                'required' => false,
+                'label' => 'WYSIWYG custom options',
+                'attr' => [
+                    'rows' => 8,
+                ]
+            ]);
+
+        if ($circleObject = $this->container->getParameter('ems_core.circles_object')) {
+            $form->add('circles', ObjectPickerType::class, [
+                'multiple' => TRUE,
+                'type' => $circleObject,
+                'dynamicLoading' => false
+
+            ]);
+        }
+
+        $form = $form->add('roles', ChoiceType::class, array('choices' => $this->getUserService()->getExistingRoles(),
+            'label' => 'Roles',
+            'expanded' => true,
+            'multiple' => true,
+            'mapped' => true,))
+            ->add ( 'create', SubmitEmsType::class, [
+                'attr' => [
+                    'class' => 'btn-primary btn-sm '
+                ],
+                'icon' => 'fa fa-plus',
+            ] )
+            ->getForm();
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            /** @var $userManager \FOS\UserBundle\Model\UserManagerInterface */
+            $userManager = $this->get('fos_user.user_manager');
+
+            $continue = TRUE;
+            $continue = $this->userExist($user, 'add', $form);
```

Comment:
userExist should allways return bool, make TRUE => true, FALSE => false

Reply:
Was allready done, but I removed the line 121 now